WTWSMS/decisions/VIET_misc_decisions.txt

```diff
@@ -93,7 +93,6 @@ decisions = {
 		potential = {
 			is_ruler = yes
 			independent = yes
-			lower_tier_than = KING
```

Comment:
No. It makes no sense that a Pagan king can't adopt local  beliefs simply because he is a king.